test/proposal.test.ts

```diff
@@ -157,7 +157,7 @@ describe('Proposal details', () => {
     const proposal = (await pod.getProposals())[0];
 
     expect(proposal.id).toBe(1);
-    expect(proposal.status).toBe('active');
+    expect(proposal.status).toBe('executed');
```

Comment:
I don't think this should have changed either, but I think one of my earlier comments will address this